lib/signature.js

```diff
@@ -21,6 +21,15 @@ const {
   purposes: {AssertionProofPurpose}
 } = jsigs;
 
+exports.wrapKeyPair = async ({ledgerNodeId, peerId}) => {
```

Comment:
Let's call this `getKeyPair` instead -- since it isn't wrapping a key pair, it's producing one. Also, this function is called in a number of places in this PR without passing `ledgerNodeId`. This indicates we are probably missing some important tests. Furthermore, `_localPeers.getKeyPair` looks like it takes `ledgerNodeId`, not `peerId`:

https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/master/lib/localPeers.js#L97

And that it only returns the key pair for a local ledger node. So I think there's some kind of conflation going on that needs to get sorted.

Reply:
function was renamed, however:

https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/52ac4a1b48e618df0cc649833b1033d41e32a657/lib/client.js#L123-L142

`_client.notifyPeer` only takes in 2 params `{localPeerId, remotePeer}` so it does not immediately at least know the ledgerNodeId (unless I'm missing something). So I used the `localPeerId` to get the key material.

Then in `./lib/localPeers` https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/52ac4a1b48e618df0cc649833b1033d41e32a657/lib/localPeers.js#L267-L278

the private method `_getKeyPair` does take in a `peerId` and as it turns out, ledgerNodeId is not necessary to find the relevant key material. So I added `peerId` as a param of `localPeers.getKeyPair`.

How should I handle this if I'm supposed to not use the `localPeerId`? Find the associated ledger node using: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/52ac4a1b48e618df0cc649833b1033d41e32a657/lib/localPeers.js#L112-L117

perhaps?

Reply:
So we should modify `sendNotification` so it requires the `ledgerNodeId`. Only a `Worker` instance calls it, so it can pass it through.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/32905e6cd329fbbdf09a4181178cad561d9c1c5a
and https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/7c1fd0d1925a8caae15c8a0e7e2ab0ec52de4c4e